backend/api/Migrations/20230515114338_InitialCreatePostgreSql.cs

```diff
@@ -0,0 +1,523 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Api.Migrations
+{
+    /// <inheritdoc />
+    public partial class InitialCreatePostgreSql : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            // MANUALLY ADDED:
+            // Adding timescale extension to the database
+            migrationBuilder.Sql("CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;");
+            //
```

Comment:
Can we add this to the README in case we need to re-make the database from InitialCreate. Also other MANUALLY ADDED lines in the migrations

Reply:
Will do

Reply:
Fixed in last commit